Lib/test/test_array.py

```diff
@@ -40,6 +40,12 @@ def test_bad_constructor(self):
         self.assertRaises(TypeError, array.array, 'xx')
         self.assertRaises(ValueError, array.array, 'x')
 
+    @support.cpython_only
+    def test_uninitialised_new(self):
+        # Prevent heap types from being created uninitialised (bpo-43916)
+        with self.assertRaises(TypeError):
+            type(iter(array.array('I')))()
```

Comment:
Can you get the type outside the assertRaises(), to ensure that the TypeError is not raised by the code getting the type? Same remark for other tests.

Reply:
Good point. I'll fix those. Thanks!

Reply:
Fixed in 4ac3c394d8ff7167ab17a096c99f3b824c5ad98a